README.md

```diff
@@ -1,2 +1,213 @@
+
+    This may prompt you to enter the username and password for signing into  Azure Portal App.
+
+    Once the username password pair successfully entered this will give a URL like follows on the browser address bar.
+
+    https://login.microsoftonline.com/common/oauth2/nativeclient?code=M95780001-0fb3-d138-6aa2-0be59d402f32
+
+    Copy the code parameter (M95780001-0fb3-d138-6aa2-0be59d402f32 in the above example) and in a new terminal enter the following CURL command with replacing the \<MS_CODE> with the code received from the above step. The \<MS_CLIENT_ID> and \<MS_CLIENT_SECRET> parameters are the same as above.
+
+    ```
+    curl -X POST --header "Content-Type: application/x-www-form-urlencoded" --header "Host:login.microsoftonline.com" -d "client_id=<MS_CLIENT_ID>&client_secret=<MS_CLIENT_SECRET>&grant_type=authorization_code&redirect_uri=https://login.microsoftonline.com/common/oauth2/nativeclient&code=<MS_CODE>&scope=Files.ReadWrite openid User.Read Mail.Send Mail.ReadWrite offline_access" https://login.microsoftonline.com/common/oauth2/v2.0/token
+    ```
+
+    The above CURL command should result in a response as follows,
+    ```
+    {
+    "token_type": "Bearer",
+    "scope": "Files.ReadWrite openid User.Read Mail.Send Mail.ReadWrite",
+    "expires_in": 3600,
+    "ext_expires_in": 3600,
+    "access_token": "<MS_ACCESS_TOKEN>",
+    "refresh_token": "<MS_REFRESH_TOKEN>",
+    "id_token": "<ID_TOKEN>"
+    }
+    ```
+
+    Set the path to your Ballerina distribution's trust store in the <TURST_STORE_PATH>. This is by default located in the following path.
+
+    $BALLERINA_HOME/distributions/jballerina-<BALLERINA_VERSION>/bre/security/ballerinaTruststore.p12
+
+    The default TRUST_STORE_PASSWORD is set to "ballerina".
+
+    WORK_BOOK_NAME, WORK_SHEET_NAME, and TABLE_NAME corresponds to workbook file name (without the .xlsx extension), worksheet name, and the table name respectively. Makesure you create a workbook with the same WORK_BOOK_NAME on Microsoft OneDrive before using the connector.
+
+3. Create a new Ballerina project by executing the following command.
+
+	```shell
+	<PROJECT_ROOT_DIRECTORY>$ ballerina init
+	```
+
+4. Import the Microsoft Graph connector to your Ballerina program as follows. The following sample program creates a new worksheet on an existing workbook on Microsoft OneDrive. Prior running this application please create a workbook on your Microsoft OneDrive account
+having the name "MyShop.xlsx". There needs to be at least one worksheet (i.e., a tab) on the workbook for this sample program to work. Note that the sample application first tries to delete an existing worksheet named "Sales" from the workbook. If its not available it may throw an error and continue executing the rest of the program. This error may get thrown during the very first round of running the sample application. Makesure, you keep the ballerina.conf file with the above mentioned configuration information before running the
```

Comment:
```suggestion
with the name "MyShop.xlsx". There needs to be at least one worksheet (i.e., a tab) on the workbook for this sample program to work. 
>**Note:** The sample application first tries to delete an existing worksheet named "Sales" from the workbook. If its not available, it may throw an error and continue executing the rest of the program. This error will get thrown during the very first round of running the sample application. Make sure that you keep the `ballerina.conf` file with the above-mentioned configuration information before running the
```

Reply:
We need not to format the line 102 as a note. It is not supposed to be a note. Hence I removed the ">**Note:** " from the line 102 and updated the rest of the text.